python/cugraph/cugraph/tests/data_store/test_gnn_feat_storage_wholegraph.py

```diff
@@ -66,6 +68,9 @@ def runtest(rank: int, world_size: int):
 @pytest.mark.skipif(
     isinstance(pylibwholegraph, MissingModule), reason="wholegraph not available"
 )
+@pytest.mark.skipif(
+    cudart.cudaRuntimeGetVersion()[1] < 11080, reason="not compatible with CUDA < 11.8"
```

Comment:
Despite the name, this is not a reliable way to get the CUDA Runtime version. It gets the runtime version used to build cuda-python, more-or-less. See the below links for some details.

https://github.com/NVIDIA/cuda-python/issues/16
https://github.com/rapidsai/rmm/issues/948
https://github.com/rapidsai/rmm/pull/1675 

Meanwhile, the CUDA 11.4 jobs [actually use cuda-python 11.8](https://github.com/rapidsai/cugraph/actions/runs/12178064576/job/33967234457#step:8:7090) -- which is fine, as it is okay to use a newer cuda-python with an older CTK -- but it means we can't use this function to get the info you're wanting. It will report 11.8 even in the 11.4 CI jobs.

The best workaround I know of is from [this PR](https://github.com/rapidsai/rmm/pull/946):
```python
import numba.cuda

def get_cudart_version():
    major, minor = numba.cuda.runtime.get_version()
    return major * 1000 + minor * 10
```

Reply:
Ack I knew about this but totally just forgot while I recommended this solution. It looks like we now have [`getLocalRuntimeVersion`](https://github.com/NVIDIA/cuda-python/issues/16#issuecomment-1774395711) now though?

Reply:
This should be resolved now

Reply:
getLocalRuntimeVersion only works correctly with the latest release of cuda-python. We have an upper bound that excludes that in 24.12.